common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -165,6 +171,12 @@ export interface ISequencedDocumentMessage {
      * @alpha
      */
     expHash1?: string;
+
+    /**
+     * The compression algorithm that was used to compress this op.
```

Comment:
I'd say not "this op" but "contents of this op". It's a bit clearer what is compressed.
Maybe we should call it contentCompression, but I hate all these longs strings - op envelope is already over 200 bytes!